candi/bashible/common-steps/node-group/090_node_users.sh.tpl

```diff
@@ -45,13 +45,12 @@ function put_user_ssh_key() {
   local main_group="$3"
   local ssh_keys="$4"
   local ssh_dir="$base_path/$user_name/.ssh"
+  local ssh_new_keys=`sed "s/\;/\n/g" <<< "$ssh_keys" | sort -u`
+  local ssh_curent_keys=`cat $ssh_dir/authorized_keys`
```

Comment:
```suggestion
  local ssh_curent_keys="$(cat $ssh_dir/authorized_keys | sort -u)"
```

Reply:
That is a problem, we use bash pipefail flag and 
```
local ssh_curent_keys=`cat $ssh_dir/authorized_keys`
```
will fail if dir doesn't exist.

Reply:
`sort -u`  is unnecessary because we write the contents of this file from a neighboring variable.